make stop and start doesn't work

YUSHANKA-M-H0YZ:volplugin yushanka$ **make stop**
vagrant destroy -f
==> mon2: VM not created. Moving on...
==> mon1: VM not created. Moving on...
==> mon0: Destroying VM and associated drives...
make clean
rm -f *.vdi
rm -f .vagrant/*.vmdk
YUSHANKA-M-H0YZ:volplugin yushanka$ **make start**
vagrant hits the file descriptor limit on OSX when running this task
10240 is the max you can set on OSX and should be higher than the default on every other OS
ulimit -n 10240; \
	if [ "x" = "x" ]; then vagrant up; else vagrant up --provider=; fi
Bringing machine 'mon0' up with 'virtualbox' provider...
Bringing machine 'mon1' up with 'virtualbox' provider...
Bringing machine 'mon2' up with 'virtualbox' provider...
==> mon0: Cloning VM...
==> mon0: Matching MAC address for NAT networking...
==> mon0: Checking if box 'contiv/centos72' is up to date...
==> mon0: Setting the name of the VM: volplugin_mon0_1466805859022_37658
==> mon0: Clearing any previously set network interfaces...
==> mon0: Preparing network interfaces based on configuration...
    mon0: Adapter 1: nat
    mon0: Adapter 2: intnet
    mon0: Adapter 3: intnet
==> mon0: Forwarding ports...
    mon0: 22 (guest) => 2222 (host) (adapter 1)
==> mon0: Running 'pre-boot' VM customizations...
A customization command failed:

["createhd", "--filename", "docker-0", "--size", "15000"]

The following error was experienced:

#<Vagrant::Errors::VBoxManageError: There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["createhd", "--filename", "docker-0", "--size", "15000"]

Stderr: 0%...
Progress state: VBOX_E_FILE_ERROR
VBoxManage: error: Failed to create medium
VBoxManage: error: Could not create the medium storage unit '/Users/yushanka/work/src/github.com/contiv/volplugin/docker-0.vdi'.
VBoxManage: error: VDI: cannot create image '/Users/yushanka/work/src/github.com/contiv/volplugin/docker-0.vdi' (VERR_ALREADY_EXISTS)
VBoxManage: error: Details: code VBOX_E_FILE_ERROR (0x80bb0004), component MediumWrap, interface IMedium
VBoxManage: error: Context: "RTEXITCODE handleCreateMedium(HandlerArg*)" at line 449 of file VBoxManageDisk.cpp
>

Please fix this customization and try again.
make: *** [start] Error 1
YUSHANKA-M-H0YZ:volplugin yushanka$

--- a/Makefile
+++ b/Makefile
@@ -26,6 +26,12 @@ clean:
 	rm -f *.vdi
 	rm -f .vagrant/*.vmdk
 
+clean-vms:
+	@echo DO NOT USE THIS COMMAND UNLESS YOU ABSOLUTELY HAVE TO. PRESS CTRL-C NOW.
+	@sleep 20
+	for i in $$(vboxmanage list vms | grep volplugin | awk '{ print $$2 }'); do vboxmanage controlvm "$$i" poweroff; vboxmanage unregistervm "$$i"; done
+	make clean
+
 update:
 	vagrant box update || exit 0
 

